schema/client-message.proto

```diff
@@ -26,9 +27,10 @@ message ClientMessage {
     GetFullOrderHistory get_full_order_history = 11;
     GetFullTradeHistory get_full_trade_history = 12;
     Redeem redeem = 13;
-    CreateAuction create_auction = 15;
-    SettleAuction settle_auction = 16;
-    DeleteAuction delete_auction = 17;
+    EditMarket edit_market = 15;
+    CreateAuction create_auction = 16;
+    SettleAuction settle_auction = 17;
+    DeleteAuction delete_auction = 18;
```

Comment:
Changing field numbers can cause issues with backwards compatibility, generally never do it and just add the new field at the end